KsApi/ServiceType.swift

```diff
@@ -380,6 +380,8 @@ public protocol ServiceType {
 
   func fetchBackedProjects(cursor: String?, limit: Int?)
     -> SignalProducer<FetchProjectsEnvelope, ErrorEnvelope>
+
+  func blockUser(userId: String) -> SignalProducer<EmptyResponseEnvelope, ErrorEnvelope>
```

Comment:
So, this will be `func blockUser(input: BlockUserInput) -> SignalProducer<EmptyResponseEnvelope, ErrorEnvelope>`